.github/workflows/main.yml

```diff
@@ -0,0 +1,130 @@
+name: trading_backend-CI
+on:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - '*'
+  pull_request:
+
+jobs:
+  lint:
+    name: ubuntu-latest - Python 3.8 - lint
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8.x'
+        architecture: 'x64'
+
+    - name: Install dependencies
+      run: pip install --prefer-binary -r dev_requirements.txt -r requirements.txt
```

Comment:
I think ccxt should only be put into `dev_requirements.txt`

Reply:
absolutely !